localflavor/locale/el/LC_MESSAGES/django.po

```diff
@@ -14,7827 +14,10656 @@ msgid ""
 msgstr ""
```

Comment:
Please remove the changes to this file. The localflavors take care of updating the translation files (which are managed through Transifex).

Reply:
Ok I restored it to the previous version